src/Cake.Common/Tools/WiX/Heat/HeatSettings.cs

```diff
@@ -0,0 +1,252 @@
+using System.Collections.Generic;
+using Cake.Core.IO;
+using Cake.Core.Tooling;
+
+namespace Cake.Common.Tools.WiX.Heat
+{
+    /// <summary>
+    /// Contains settings used by <see cref="HeatRunner"/>.
+    /// </summary>
+    public sealed class HeatSettings : ToolSettings
+    {
+        /// <summary>
+        /// Gets or sets the WiX extensions to use.
+        /// </summary>
+        public IEnumerable<string> Extensions { get; set; }
+
+        /// <summary>
+        /// Gets or sets a value indicating whether [no logo].
+        /// </summary>
+        /// <value>
+        ///   <c>true</c> if [no logo]; otherwise, <c>false</c>.
+        /// </value>
+        public bool NoLogo { get; set; }
```

Comment:
@RLittlesII some whitespace in this file:

![image](https://cloud.githubusercontent.com/assets/1271146/16130549/a5652594-3401-11e6-9c38-0631c03c3a59.png)